pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -769,3 +780,7 @@ def sort_legend_elem(elem):
         value = -elem.get(geom_type, 0)
 
         return category, value
+
+    def extract_param(self, params, key, default):
+        param = params.pop(key, default)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'param' is assigned to but never used](https://app.codacy.com/manual/pyramid_oereb/pyramid_oereb/pullRequest?prid=4790103)